ezpublish/config/config.yml

```diff
@@ -50,10 +50,6 @@ assetic:
         #yui_css:
         #    jar: "%kernel.root_dir%/Resources/java/yuicompressor-2.4.7.jar"
 
-ez_publish_legacy:
-    enabled: true
-    root_dir: %kernel.root_dir%/../ezpublish_legacy
```

Comment:
Nope. `enabled` is now true by default (needs LegacyBundle to be instantiated, and `root_dir` has `%kernel.root_dir%/../ezpublish_legacy` as default value.